apm-agent-core/src/main/java/co/elastic/apm/report/serialize/DslJsonSerializer.java

```diff
@@ -85,11 +85,13 @@
     private static final byte NEW_LINE = (byte) '\n';
     private final Collection<String> excludedStackFrames = Arrays.asList("java.lang.reflect", "com.sun", "sun.", "jdk.internal.");
     static final int MAX_VALUE_LENGTH = 1024;
+    static final int MAX_SQL_VALUE_LENGTH = 10000;
```

Comment:
Rename this and other occurrences of SQL to LongString. For example `MAX_LONG_STRING_VALUE_LENGTH`, `writeLongStringField` etc.

While this is currently only used for SQLs, it will likely be used for other fields in the future.

Reply:
Accepted. I'll change it.